docs/deployment/manage-projects.md

```diff
@@ -13,12 +13,12 @@ tags:
 View, sort, and tag your projects through the **Projects** page. Refer to this page to **manage** and **troubleshoot** thousands of repositories by identifying scan issues or scans with a high number of findings.
 
 :::note What is a project?
-A **project** is any repository that you have scanned with Semgrep AppSec Platform through CI or Semgrep Managed Scans. It includes that repository's findings and metadata.
+A **project** is a repository, or part of a repository, that you can scan through Semgrep AppSec Platform, either using CI or Semgrep Managed Scans. Its scans can be viewed on the **Project details** page, and its findings can be viewed on the individual Semgrep products' **Finding** pages.
```

Comment:
I'd probably leave off the "can" in "can scan", but I'm open to thoughts on why you're adding that.

Unfortunately the "either using CI or Semgrep Managed Scans" is not accurate either, since you can send local (CLI, non-CI) scan results to the platform too. I don't know how to express this compactly 😓

Reply:
I'll simplify "can scan" since I think its presence in Platform indicates that the repo can be scanned by Semgrep

What about something like `you scan through Semgrep AppSec Platform, either using CI or Semgrep Managed Scans. This also includes local CLI scans whose results you have sent for viewing on Semgrep AppSec Platform`?